apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -1413,18 +1413,17 @@ def update_obs(
     type for the column, the entire update will raise a ``ValueError`` exception.
 
     Args:
-        exp: The :class:`SOMAExperiment` whose ``obs`` is to be updated. Must
-        be opened for write.
+        exp: The :class:`SOMAExperiment` whose ``obs`` is to be updated. Must be opened for write.
 
         new_data: a Pandas dataframe with the desired contents.
 
         context: Optional :class:`SOMATileDBContext` containing storage parameters, etc.
 
-        platform_config: Platform-specific options used to update this array, provided
-        in the form ``{"tiledb": {"create": {"dataframe_dim_zstd_level": 7}}}``
+        platform_config: Platform-specific options used to update this array, provided in the form
+            ``{"tiledb": {"create": {"dataframe_dim_zstd_level": 7}}}``
```

Comment:
Sphinx grossly misrenders this (even moving words out of the order they were typed here!!!!!) unless there's this indent. Weird but real.